src/app/layout/Layout.tsx

```diff
@@ -17,7 +18,11 @@ export function Layout({ children }: { children: React.ReactNode }) {
       {(pathname !== "/login" || isSidebarOpen) && (
         <Sidebar isSidebarOpen={isSidebarOpen} toggleSidebar={toggleSidebar} />
       )}
-      <div className="flex flex-col w-full h-screen lg:ml-48">
+      <div
+        className={cx(
```

Comment:
@muhammad-bilal-ali-khokhar  why we need to use `cx`?

Reply:
@nadirhussain786  `cx` using for conditional classes